bevy_kayak_renderer/src/render/unified/shader.wgsl

```diff
@@ -7,7 +7,11 @@ var<uniform> view: View;
 
 struct QuadType {
     t: i32,
+    _padding_1: i32,
```

Comment:
Just an informative comment. Because `QuadType` is not in an array we don't actually need to specify the padding in the shader. I think it provides clarity though and matches the CPU side so we don't need to change this.

Reply:
I see! Thanks for the info, it's useful to know